src/frontend/Komponenter/Behandling/SettPåVent/LokalkontorOppgavevalg.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, Dispatch, SetStateAction, useState } from 'react';
+import { CheckboxGroup, Checkbox, BodyShort } from '@navikt/ds-react';
+import styled from 'styled-components';
+import { dagensDatoFormatert, formaterIsoDato } from '../../../App/utils/formatter';
+
+const FlexBox = styled.div`
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+`;
+
+export type SendtOppgave = {
+    vurderHenvendelseOppgave: VurderHenvendelseOppgavetype;
+    datoOpprettet: string;
+};
+
+type Props = {
+    aktuelleOppgaver: VurderHenvendelseOppgavetype[];
+    sendteOppgaver: SendtOppgave[];
+    settOppgaverMotLokalkontor: Dispatch<SetStateAction<VurderHenvendelseOppgavetype[]>>;
+    oppgaverMotLokalkontor: VurderHenvendelseOppgavetype[];
+    erBehandlingPåVent: boolean;
+};
+
+export const LokalkontorOppgavevalg: FC<Props> = ({
+    aktuelleOppgaver,
+    sendteOppgaver,
+    settOppgaverMotLokalkontor,
+    oppgaverMotLokalkontor,
+    erBehandlingPåVent,
+}) => {
+    const tidligereSendteLokalkontorOppgaver = sendteOppgaver.map((oppgave) => {
+        return oppgave.vurderHenvendelseOppgave;
+    });
+
+    const [checkedOppgaver, settCheckedOppgaver] = useState([
+        ...tidligereSendteLokalkontorOppgaver,
+        ...oppgaverMotLokalkontor,
+    ]);
+    const erSendt = (oppgave: VurderHenvendelseOppgavetype): boolean =>
+        tidligereSendteLokalkontorOppgaver.includes(oppgave);
+    const erValgt = (oppgave: VurderHenvendelseOppgavetype): boolean => {
+        return oppgaverMotLokalkontor.includes(oppgave);
+    };
+
+    const lagOppgaveSendtTekst = (oppgave: VurderHenvendelseOppgavetype): string | undefined => {
+        const matchedOppgavestatus = sendteOppgaver.find(
+            (status) => status.vurderHenvendelseOppgave === oppgave
+        );
+
+        if (erValgt(oppgave) && erBehandlingPåVent) {
+            return '(Oppgave er sendt ' + dagensDatoFormatert() + ')';
+        }
+        return matchedOppgavestatus?.datoOpprettet
+            ? ' (Oppgave sendt ' + formaterIsoDato(matchedOppgavestatus?.datoOpprettet) + ')'
+            : undefined;
+    };
+
+    const filtrerTidligereSendteOppgaver = (
+        oppgaver: VurderHenvendelseOppgavetype[]
+    ): VurderHenvendelseOppgavetype[] => {
+        return oppgaver.filter((oppgave) => !tidligereSendteLokalkontorOppgaver.includes(oppgave));
+    };
+
+    return (
+        <div>
+            <CheckboxGroup
+                legend="Send oppgave til lokalkontoret"
+                onChange={(oppgaver) => {
+                    settOppgaverMotLokalkontor(filtrerTidligereSendteOppgaver(oppgaver));
+                    settCheckedOppgaver(oppgaver);
+                }}
+                size="small"
+                value={checkedOppgaver}
+            >
+                {aktuelleOppgaver.map((oppgave) => (
+                    <FlexBox>
```

Comment:
Når vi rendrer lister må vi sende med key til DOM'et, eller får vi feil i consolen 🤓 

https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

Reply:
Godt poeng!